src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationAndGeneratorDriverTranslationAction_Actions.cs

```diff
@@ -36,24 +35,17 @@ public override Task<Compilation> TransformCompilationAsync(Compilation oldCompi
 
             internal sealed class TouchAdditionalDocumentAction : CompilationAndGeneratorDriverTranslationAction
             {
+#pragma warning disable IDE0052 // Remove unread private members
```

Comment:
To explain a bit of the history of the code here: the source generator API from the compiler originally had an API for doing incremental updates; that was hastily commented out in a preview release, and as a result the TransformGeneratorDriver functions weren't doing anything. I've removed the TransformGeneratorDriver code for now, but am leaving the additional actions in place because there's already been a certain amount of investment in creating these actions correctly in all of the various calls where we fork snapshots. Since the expectation is the compiler API will get worked on soonish, I'm going to leave the actions in place for now.